Sources/ArcGISToolkit/Components/FeatureFormView/EmbeddedFeatureFormView.swift

```diff
@@ -87,37 +104,23 @@ extension EmbeddedFeatureFormView {
                     formViewModel: embeddedFeatureFormViewModel
                 )
             case let element as FieldFormElement:
-                makeFieldElement(element)
+                if !(element.input is UnsupportedFormInput) {
+                    FieldFormElementView(element: element)
+                }
```

Comment:
You could use a `where` clause here if you want:

```swift
case let element as FieldFormElement where !(element.input is UnsupportedFormInput):
    FieldFormElementView(element: element)
```

That way there's only one pathway that results in `EmptyView`.